core/server/models/settings.js

```diff
@@ -81,7 +81,7 @@ Settings = ghostBookshelf.Model.extend({
         }).then(function () {
             var themeName = setting.value || '';
 
-            if (setting.key !== 'activeTheme') {
+            if (setting.key !== 'activeTheme' || options.importing) {
```

Comment:
Regarding `validation.validateActiveTheme(themeName)`:
1. The naming of the function is totally confusing. This function checks if the activeTheme value in the settings table exists on file system.
2. In future thinking, i think this extra check here can be deleted, because it depends on which storage adapter you are using.
3. When Ghost starts and your theme is lost, it does not help. It get's only triggered when we update `activeTheme`. 

---

#### So here is my suggestion
We can merge the PR for `LTS`. 

But delete validation.validateActiveTheme(themeName) completely for `1.0` and go with the risk that if somebody activates a theme in admin, he won't get an error message because theme is not available. Later we can re-add a check which does `storageAdapter.exists`, but i am not 100% sure that this is very important, because when navigating to the blog, we would directly see that the `activeTheme` is not available. 

https://github.com/TryGhost/Ghost/blob/master/core/server/middleware/theme-handler.js#L92
This seems to have a bug.
When navigating to `localhost:2368`, i'll get an hbs error. Might be related to https://github.com/TryGhost/Ghost/issues/6948. But we can just add `blogApp.engine('hbs', hbs.express3());` for now. Small PR?